### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by RachelBerman**

### CODE REVIEW

File: lesson-3/Berman-lesson3/About/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  </head>
+  <body>
+
+
+    <h1>About</h1>
+
+    <img src='../images/rachel.jpg'width='300px'/>
+
+    <ul>
+      <li><a href="../../Berman-lesson3/">Home</a></li>
```
Comment (vendelal) [inline lesson-3/Berman-lesson3/About/index.html]: So close! You went one step too far back here. You only need one `../` to get back to the main folder. Your main folder in this case is `Berman-lesson3`.

File: lesson-3/Berman-lesson3/Architecture/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  </head>
+  <body>
+
+
+    <h1>Architecture</h1>
+
+    <img src='../images/renders.jpg'width='300px'/><br/>
+
+    <ul>
+      <li><a href="../../Berman-lesson3/">Home</a></li>
```
Comment (vendelal) [inline lesson-3/Berman-lesson3/Architecture/index.html]: Same comment as above.

File: lesson-3/Berman-lesson3/Art/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  </head>
+  <body>
+
+
+    <h1>Art</h1>
+
+    <img src='../images/drawing.jpg'width='300px'/>
+
+    <ul>
+      <li><a href="../../Berman-lesson3/">Home</a></li>
```
Comment (vendelal) [inline lesson-3/Berman-lesson3/Art/index.html]: Same comment as above

File: lesson-3/Berman-lesson3/GraphicDesign/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  </head>
+  <body>
+
+
+    <h1>Graphic Design</h1>
+
+    <img src='../images/windrose.jpg'width='300px'/>
+
+    <ul>
+      <li><a href="../../Berman-lesson3/">Home</a></li>
```
Comment (vendelal) [inline lesson-3/Berman-lesson3/GraphicDesign/index.html]: Same comment as above

File: lesson-3/Berman-lesson3/index.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+<html>
+  <head>
+    <title>Home page</title>
+  </head>
+  <body>
+
+
+    <h1>Home!</h1>
+
+    <a href="About/">Link to about page</a><br/>
```
Comment (vendelal) [inline lesson-3/Berman-lesson3/index.html]: It won't make a huge difference here, but make sure to always write your links exactly how the folders are named. In this case you named your folders with lowercase letters, but wrote them capitalized. This should be `about/`

**Review by vendelal — COMMENTED** [review]
Nice work! Just some small changes so that your links back to home will work 👍

File: lesson-3/Berman-lesson3/index.html (hunk 5 above)
Comment (RachelBerman) [reply lesson-3/Berman-lesson3/index.html]: On my computer the folders show capital letters. Does this change when i upload it here?

Comment (vendelal) [reply lesson-3/Berman-lesson3/index.html]: Odd, you're right. Not sure where I saw them lowercase haha

**Pull request closed without merging**